cmd/antrea-agent/config.go

```diff
@@ -192,3 +194,7 @@ type WireGuardConfig struct {
 	// The port for the WireGuard to receive traffic. Defaults to 51820.
 	Port int `yaml:"port,omitempty"`
 }
+
+type EgressConfig struct {
+	ExceptCIDRs []string `yaml:"exceptCIDRs,omitempty"`
```

Comment:
Could you add comment for this configuration? It could be the same as that in the conf file.

Reply:
Sure, fix it later :-)